integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java

```diff
@@ -279,86 +253,58 @@ public void testDropFunction2() { // drop
 
   @Test
   public void testDropBuiltInFunction() throws SQLException { // drop
-    try (Connection connection =
-            DriverManager.getConnection(
-                Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root", "root");
+    try (Connection connection = EnvFactory.getEnv().getConnection();
         Statement statement = connection.createStatement()) {
       try {
         statement.execute("drop function abs");
         fail();
       } catch (SQLException throwable) {
-        assertTrue(throwable.getMessage().contains("Built-in function"));
+        assertTrue(
+            throwable.getMessage().contains("Built-in function ABS can not be deregistered"));
       }
-      statement.execute("INSERT INTO root.vehicle.d1(time, s1) VALUES(1, -10.0)");
-      ResultSet rs = statement.executeQuery("SELECT ABS(s1) FROM root.vehicle.d1");
-      Assert.assertTrue(rs.next());
-      Assert.assertEquals(1, rs.getLong(1));
-      Assert.assertEquals(10.0F, rs.getFloat(2), 0.00001);
-      Assert.assertFalse(rs.next());
```

Comment:
Why delete these assert

Reply:
CI was not stable when calling insert yesterday, these are added back now.